### ISSUE TITLE
Dataset page description: design is not l10n-friendly 

### ISSUE THREAD

**Issue by bobchao** [common-voice/common-voice#1877]
https://github.com/mozilla/voice-web/blob/4ab882b4b15bb2033819d97cfbe5743fd1b81fd2/web/src/components/pages/datasets/dataset-info.css#L95

The CSS using first-line to create a line of heading text, which looks good in English with fixed width, but caused a weird line-breaking effect in some language:

English: "We are building"
![We are building](https://user-images.githubusercontent.com/168931/53781276-29505500-3f43-11e9-85a3-d6718b4b81c5.png)

Dutch: We bouwen een... (means "We are building one" -- not as good as English but okay.)
![We bouwen een](https://user-images.githubusercontent.com/168931/53781304-408f4280-3f43-11e9-90aa-4a3bc86548dc.png)

Chinese: 我們正在打造一... (means "We are building o..." -- the last char is not even finished a word) 
![我們正在打造一](https://user-images.githubusercontent.com/168931/53781289-2c4b4580-3f43-11e9-84c3-d406aa7c2f91.png)

Though localizers can try a different sentence to avoid this issue, it would be better to take this into consideration when doing design.

**Comment by Gregoor** [comment]
Thanks for raising this, I was curious to hear reports how this works in different languages. I think we should then only have this for languages where we know it makes sense. So we know it works in:
- Dutch
- German
- French
- English
- Spanish

**Comment by irvin** [comment]
Or just separate them into two strings and the local l10n team can find the most reasonable way to translate them.

**Comment by Txopi** [comment]
Actual design doesn't work with Basque, either.

**Comment by zaccolley** [comment]
Also an accessibility issue, this is not a heading. I will work on a fix for this.
